config/runtime/watchdog-reset.jinja2

```diff
@@ -0,0 +1,4 @@
+{%- set boot_commands = 'ramdisk' %}
+{%- set test_method = 'watchdog-reset' %}
+{%- set base_template = 'base/' + runtime + '.jinja2' %}
+{%- extends base_template %}
```

Comment:
This is fine for now, but I'd suggest factoring this into a single generic test template at some point: with job params such as `test_method` (and potentially `boot_commands` as well) this shouldn't be too difficult and would allow us to reduce the number of templates (at least `kselftest`, `sleep` and `v4l2-decoder-conformance` would benefit from it, in addition to this one, maybe even `baseline` can be made to work that way).

Reply:
Makes sense. I can tackle this as a separate PR after this gets merged.